docs/proposals/20210310-opt-in-autoscaling-from-zero.md

```diff
@@ -188,6 +195,40 @@ type DockerMachineTemplate struct {
 ```
 _Note: the `ResourceList` and `ResourceName` referenced are from k8s.io/api/core/v1`_
 
+`platform.NodeInfo` is a struct that contains the architecture and operating system information of the node, to implement 
+in the providers integration code.
+Its definition would look like this:
+
+```go
+package platform
```

Comment:
Just to double check, are we sure we want to expose those types in a non-versioned package?
Also, given this is going to be part of the contract, shouldn't we use a package name that enforce this link (and that we can eventually expand with other contract related stuff)?

Reply:
We were considering not to have these types in the cluster api repo, but delegating implementations to define their based on the contract defined here